examples/fem_system/fem_system_ex1/fem_system_ex1.C

```diff
@@ -36,6 +36,8 @@
 #include "libmesh/exodusII_io.h"
 #include "libmesh/kelly_error_estimator.h"
 #include "libmesh/mesh.h"
+#include "libmesh/serial_mesh.h"
```

Comment:
Could you switch this to replicated_mesh.h (and later to ReplicatedMesh)?  The old misleading names are deprecated.